src/App.jsx

```diff
@@ -0,0 +1,74 @@
+import 'bulma/css/bulma.css';
+import './App.scss';
+import { useState } from 'react';
+
+export const goods = [
+  'Dumplings',
+  'Carrot',
+  'Eggs',
+  'Ice cream',
+  'Apple',
+  'Bread',
+  'Fish',
+  'Honey',
+  'Jam',
+  'Garlic',
+];
+
+export const App = () => {
+  const [selectedGood, setValue] = useState('Jam');
```

Comment:
1) You've already had the 'Jam' in the goods array.
2)  It's recommended to name the setter function based on the specific state variable it's associated with
```suggestion
  const [selectedGood, setSelectedGood] = useState(goods[8]);
```